src/lib/apollo/initApollo.js

```diff
@@ -14,7 +14,8 @@ const create = (initialState) =>
     connectToDevTools: process.browser,
     ssrMode: !process.browser,
     link: new HttpLink({
-      uri: "http://localhost:3030/graphql-alpha",
+      uri: `${process.env.PROTOCOL}://${process.env.HOST}:${process.env.PORT}/${process.env
+       .ENDPOINT}`,
```

Comment:
I would destructure these (including METEOR_TOKEN) from `process.env` toward the top of the file and check each one throwing a specific error if it isn't set. That way when someone inevitably starts it up with improper configuration they'll get a more helpful error message like "You must set PROTOCOL env variable".

I'm also not sure why it would be necessary to break this up rather than just having a single `GRAPHQL_URL` variable.